app/modules/transcribe/modals/transcribe-next.controller.js

```diff
@@ -42,4 +43,32 @@ function TranscribeNextController($modalInstance, ClassificationFactory) {
             .then(close, error);
     }
 
+    function viewAnnotations() {
+        if (annotations.length > 0) {
+            var i, doc = new jsPDF('p', 'mm');
+            for (i = 0; i < annotations.length; i++) {
```

Comment:
I used `for` because `forEach` has been deprecated https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for_each...in.
You're right. I can get rid of `map`.